api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageQueueDAO.java

```diff
@@ -29,21 +30,26 @@
 
     @Override
     public void sendMessage(MessagePayload messagePayload, MessageType messageType) {
+        sendMessageWithDelay(messagePayload, messageType, Period.ofDays(0));
```

Comment:
Might be worth creating a constant `NO_DELAY = Period.ofDays(0);` and use it here rather than creating a new Period every time.

Reply:
Updated to use Period.ZERO